package.json

```diff
@@ -2,8 +2,30 @@
   "name": "radiance-ui",
   "version": "17.2.3",
   "description": "Curology's React based component library",
-  "main": "dist/bundle.js",
-  "module": "dist/bundle-es/index.js",
+  "main": "./dist/cjs/index.cjs",
+  "module": "./dist/esm/index.js",
+  "exports": {
+    ".": {
+      "browser": "./dist/esm/index.js",
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.cjs"
+    },
+    "./lib/constants": {
+      "browser": "./dist/esm/constants/index.js",
+      "import": "./dist/esm/constants/index.js",
```

Comment:
Should we move the changes in #826 in here as well so that this functionality is more "complete" or do we want two separate PRs to maintain the distinction?

Reply:
The approach in this PR makes #826 unnecessary.

With the current setup the common.js version is a single file bundle, so the only way to include constants is to add them in the main index, which is what #826 does.

With the rollup changes in this PR we create multiple entrypoints for common.js and ES modules, so it's not necessary to add constants to the main index; we can just create a separate common.js module for them instead.

I think this approach is preferable since it allows you to import things in the same exact way regardless of module system, and you get to keep each bundle small.